src/Query.ts

```diff
@@ -217,6 +227,16 @@ export class Query {
         }
     }
 
+    private readonly sortByRegexp = /sort by (status|due|done)/;
```

Comment:
I prefer all regexes at the top to see at first glance what's possible.

Please add the caret at the beginning to prevent mix ups :)
```suggestion
    private readonly sortByRegexp = /^sort by (status|due|done)/;
```

Reply:
#1 fixed in ca3d6dc

Re caret: the other regexes don't have a caret, and I don't think it's a problem to not have it b/c it does line-based matching anyway.
https://github.com/schemar/obsidian-tasks/blob/ca3d6dc496c4c2f58b3cc270ac082c3a175f60cb/src/Query.ts#L68-L68

I'd prefer to keep the caret usage consistent amongst the regexes, otherwise I'd be confused as to why one of them uses it and the others don't

Reply:
Forget what I said about carets, I just merged in `main`. There they are ^^

Reply:
49050bc